python/ray/util/scheduling_strategies.py

```diff
@@ -11,7 +11,7 @@
 # "SPREAD": Spread scheduling on a best effort basis.
 
 
-@PublicAPI(stability="beta")
+@PublicAPI
```

Comment:
We should make sure this is written to release notes

Reply:
Yes. They should have been marked as stable a while back.